docker/scripts/workspace-entrypoint.sh

```diff
@@ -48,14 +42,11 @@ PLATFORM="$(uname -m)"
 # Make sure the user has the correct permissions
 sudo chown -R 1000:1000 /workspaces/isaac_ros-dev/install
 
-# Build ROS dependencyS
-
-echo "source /opt/ros/${ROS_DISTRO}/setup.bash" >> ~/.bashrc
-
+# Source ROS2
 source /opt/ros/${ROS_DISTRO}/setup.bash
+echo "source /opt/ros/${ROS_DISTRO}/setup.bash" >> ~/.bashrc
```

Comment:
I was testing on machine yesterday and still had to manually source ROS with this commit running. I'd double check that this works